docs/src/pages/components/app-bar/app-bar.md

```diff
@@ -46,17 +46,16 @@ You can use the `useScrollTrigger()` hook to respond to user scroll actions.
 ## Placement
```

Comment:
The position of the header is wrong. It's in between the scrolling section.

Reply:
I'll move up it above the Scrolling section.